### ISSUE TITLE
Responsive Web Design Projects - Build a Product Landing Page - Email submit to static page not working.

### ISSUE THREAD

**Issue by abhilahn9** [freeCodeCamp/testable-projects-fcc#930]
#### Browser Information

* Browser Name, Version: Google Chrome Version 75.0.3770.142 (Official Build) (64-bit)
* Operating System: Windows 10 Professional
* Mobile, Desktop, or Tablet: Desktop

#### Your Code / Link to Your Pen

```
https://codepen.io/abhilashn9/pen/jjeMML?editors=1100
 
 
 
```

**Comment by petercr** [comment]
@abhilahn9 So I took a look at your CodePen and I was able to figure out why you were not passing the last test. 😄 

If you look down at the first line in the box which gives you details about the error, it says `The #email input should have a name attribute : expected false to equal true`

What that means is that the tests are looking for the input `#email` to have an `name` _attribute_, just like it has a `id` attribute that equals `email`.  So if you just add `name="email"` to that input `#email` then your project will pass all the tests. 👍 

```html
<input type="email" name="email" id="email"  placeholder="Enter your E-Mail">
```

![screenshot of passing the tests](https://user-images.githubusercontent.com/26460352/61495157-2dc35a80-a986-11e9-988e-c0de9fc675a6.jpg)
